website/versions/source-tailscale.json

```diff
@@ -0,0 +1 @@
+{ "latest": "plugins-source-tailscale-v1.0.0" }
```

Comment:
Shouldn't this be a pre `1.0.0` release as we should release it with a preview/alpha tag for feedback?

Reply:
I think we keep all the versions in the same format e.g. `v1.0.0` without any `pre` for simplicity.
We only difference is how we document the plugins in https://www.cloudquery.io/docs/plugins/sources/overview